src/index.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as v1Module from './v1';
+import * as v1p1beta1Module from './v1p1beta1';
+import * as v1p2beta1Module from './v1p2beta1';
+import * as v1p3beta1Module from './v1p3beta1';
+import * as v1p4beta1Module from './v1p4beta1';
+import * as helpers from './helpers';
+// tslint:disable-next-line no-any
+const gapic: any = Object.freeze({
```

Comment:
I don't think we need `gapic` here at all. We can just `import * as v1p1beta1 from './v1p1beta1'` and then export them directly.

Also, I would replace the `for` loop with 5 `Object.assign` lines there (it not a usual advice I'd give, but in this particular case, I think making assignments explicit is better).

Reply:
cool, made changes, thanks

Reply:
Seems if we directly do like this, it won't work: @alexander-fenster 
```
import * as v1 from './v1';
object.assign(v1.ImageClient.prototype, helpers.call('v1'));
export {v1}
// v1 is not changed at all, no feature methods are in this module.
```

Reply:
talked in person. We need to do _declaration merging_ like [here](https://github.com/googleapis/nodejs-speech/blob/master/src/v1/speech_client.ts#L477-L478).

Reply:
exported interface of feature method and extends the methods in client file. Thanks